sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationPlugin.kt

```diff
@@ -22,4 +22,9 @@ interface AnnotationPlugin : MapPlugin, MapSizePlugin, MapStyleObserverPlugin {
     type: AnnotationType,
     annotationConfig: AnnotationConfig?
   ): AnnotationManager<*, *, *, *, *, *, *>
+
+  /**
+   * Remove an annotation manger that created before, this will remove the inside layer and source too.
```

Comment:
suggest to rephrase as `Removes an annotation manager, this will remove the underlying layer and source from the style.`